addons/gut/awaiter.gd

```diff
@@ -22,6 +24,10 @@ func _physics_process(delta):
 		if(_elapsed_frames >= _wait_frames):
 			_end_wait()
 
+	var is_waiting_for_object_to_be_freed := hash(_object_waiting_to_be_freed) != hash(null)
```

Comment:
Why the use of `hash` when checking for `null`?

Reply:
With hindsight it does look weird. I probably hit a case where I needed to differentiate null and freed objects like mentioned [in this forum post](https://forum.godotengine.org/t/distinguish-between-freed-object-and-null/3838/2).

If `_object_waiting_to_be_freed != null` still passes all tests, it would sound simpler. Otherwise documenting why it has to be weird will help future readers!